requirements.txt

```diff
@@ -1,15 +1,8 @@
 GitPython>=2.0.0
-Pint>=0.16.0,<0.17.0
 PyYAML>=5.4.1,<6.0.0
 backoff>=1.10.0,<2.0.0
-boto3>=1.16.34,<2.0.0
 click>=8.0.0
-docker>=4.4.0,<5.0.0
-ipython>=5.0.0
-kubernetes>=12.0.1,<13.0.0
 packaging>=21.3
-pyOpenSSL>=20.0.0
 pypng>=0.0.20
 requests>=2.25.0,<3.0.0
-rsa>=4.7,<5.0.0
```

Comment:
@dwanderson-intel any idea why this was in here?

Reply:
dunno, looks like it came over in the large, 8.0.0 runs release. Could've been bundled as part of cluster management?